src/main/webapp/WEB-INF/view/login.jsp

```diff
@@ -18,6 +18,12 @@
 <head>
   <title>Login</title>
   <link rel="stylesheet" href="/css/main.css">
+  <style>
```

Comment:
Did project 0 ask you to put this CSS inline here? Generally it's better practice to put this in a css file and link to it with a <link> tag. If they asked you to do it like this, we can go with it for now, though

Reply:
It was included in the code given in Project 1, but I can put it it into a separate css file, considering that might be easier for future changes as well

Reply:
Okay, if that's how it was in Project 0, I'm cool with it.